app/controllers/api/v1/facts_controller.rb

```diff
@@ -0,0 +1,70 @@
+class Api::V1::FactsController < ApplicationController
+    include AuthenticationCheck
+  
+    before_action :is_user_logged_in
+    before_action :set_fact, only: [:show, :update, :destroy]
+  
+    # GET /members/:member_id/facts
+    def index
+      @member = Member.find(params[:member_id])
+      render json: @member.facts # note that because the facts route is nested inside members
+                               # we return only the facts belonging to that member
+    end
+  
+    # GET /members/:member_id/facts/:id
+    def show
+      # your code goes here
+      if set_fact
```

Comment:
You already have `before_action :set_fact`, so you're duplicating database calls for several of these actions.